src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -49,7 +49,6 @@ public function collect(Request $request, Response $response, \Exception $except
             }
         }
 
-        $content = null;
```

Comment:
`$content` is directly assigned in the try and in the catch and there is not check on it.